python/mxnet/ndarray_doc.py

```diff
@@ -77,6 +77,15 @@ class BroadcastToDoc(NDArrayDoc):
     (2L, 2L, 2L, 3L)
     """
 
+class CustomDoc(NDArrayDoc):
+    """
+    Example
+    -------
+    Applies a custom operator named `my_custom_operator` to `input`.
+
+    >>> output = mx.symbol.Custom(op_type='my_custom_operator', data=input)
```

Comment:
Writing a custom operator here will make this example lengthy without contributing much to "how to use mxnet.ndarray.Custom?". How to write the custom operator itself is a different topic and is covered by the tutorial which is linked to from here.